src/index.js

```diff
@@ -1,42 +1,90 @@
+/* eslint-disable no-use-before-define */
 import './style.css';
 
-function showTodo(todo) {
-  const list = document.querySelector('.list');
-  const newlist = document.createElement('li');
-  newlist.classList.add('newlist');
-  newlist.innerHTML = `
-      <input id="${todo.index}" class="check" type="checkbox"/>
-      <span class="text">${todo.description}</span>
-      <button class="delete"><img src="../images/dots.svg"></button>
-    `;
-  list.append(newlist);
+const newTodoForm = document.querySelector('#new-todo-form');
+const todoList = document.querySelector('.todo-list');
+let todos = JSON.parse(localStorage.getItem('todos') || '[]');
+
+function init() {
+  newTodoForm.addEventListener('submit', newTodo);
+  updateList();
+}
+
+function updateTodos(newTodos) {
+  todos = newTodos;
+  localStorage.setItem('todos', JSON.stringify(todos));
+
+  updateList();
+}
+
+function updateList() {
+  const description = todos.map((todo) => `
+            <li class="card todo-list-item ${todo.complete ? 'completed' : ''}" data-id="${todo.id}">
+              <input type="checkbox" ${todo.complete ? 'checked' : ''} class="checkbox"/>
+              <input type="text"value="${todo.description}" class="inputtext"/>
+              <button type="button">🗑</button>
+            </li>
+          `).join('');
+
+  todoList.innerHTML = description;
+
+  const deleteButtons = todoList.querySelectorAll('.todo-list-item button');
+
+  deleteButtons.forEach((button) => button.addEventListener('click', (event) => {
+    const li = event.target.parentElement;
+    removeTodo(li.id);
+  }));
+
+  const completedCheckboxes = todoList.querySelectorAll('.checkbox');
+  completedCheckboxes.forEach((checkbox) => checkbox.addEventListener('click', toggleComplete));
 }
 
-const todoList = [];
-let id = 1;
-function add(text) {
-  const todo = {
-    description: text,
+function newTodo(e) {
+  e.preventDefault();
+
+  const newTodoText = this.querySelector('[name="new-todo-content"]');
+  const description = newTodoText.value || '';
+
+  if (description.length === 0) {
+    return;
+  }
+
+  const newTodo = {
+    description: newTodoText.value,
     completed: false,
-    index: id += 1,
+    id: todos.length,
   };
```

Comment:
- [ ] Good job so far :+1:  But one more thing was requested by the previous reviewer  :cry:  From the requirement by default, new tasks should have the property completed set to false ( `completed: false` not `completed: 0`  ) and the property index set to the value of the new array length (i.e 5th task index should be` index: 5 `  not  `index: 4` or in your case `id: 5` not `id: 4`  ), Please fix it. Check the screenshot below :point_down: 

![Screenshot from 2022-07-16 08-26-48](https://user-images.githubusercontent.com/6615394/179347134-3603aaab-9b61-4bbb-9d6a-2c8dadc43335.png)


> * [ ]   Your  property index should be  set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).
> 
> ![image](https://user-images.githubusercontent.com/43877689/179252285-10b1421c-24f0-4530-8069-7f8998a22fcb.png)